### PR TITLE
Fix 9 UI entites according to sat6.2 changes

### PR DESCRIPTION
**Pull request by abalakh**
Part of #3329 

Some tests results:
```python
py.test tests/foreman/ui/test_trend.py -k 'test_positive_update'
================================ test session starts =================================
platform linux2 -- Python 2.7.10, pytest-2.8.7, py-1.4.31, pluggy-0.3.1
rootdir: /home/andrii/workspace/robottelo, inifile: 
plugins: xdist-1.13.1, repeat-0.2
collected 3 items 

tests/foreman/ui/test_trend.py .

=================== 2 tests deselected by '-ktest_positive_update' ===================
====================== 1 passed, 2 deselected in 34.01 seconds =======================

py.test tests/foreman/ui/test_template.py -n 2
================================ test session starts =================================
platform linux2 -- Python 2.7.10, pytest-2.8.7, py-1.4.31, pluggy-0.3.1
rootdir: /home/andrii/workspace/robottelo, inifile: 
plugins: xdist-1.13.1, repeat-0.2
gw0 [11] / gw1 [11]
scheduling tests via LoadScheduling
...........
============================ 11 passed in 182.96 seconds =============================


py.test tests/foreman/ui/test_syncplan.py -k 'test_positive_create_with_sync_interval'
================================ test session starts =================================
platform linux2 -- Python 2.7.10, pytest-2.8.7, py-1.4.31, pluggy-0.3.1
rootdir: /home/andrii/workspace/robottelo, inifile: 
plugins: xdist-1.13.1, repeat-0.2
collected 19 items 

tests/foreman/ui/test_syncplan.py .

========= 18 tests deselected by '-ktest_positive_create_with_sync_interval' =========
===================== 1 passed, 18 deselected in 113.69 seconds ======================

py.test tests/foreman/ui/test_setting.py -n 2
================================ test session starts =================================
platform linux2 -- Python 2.7.10, pytest-2.8.7, py-1.4.31, pluggy-0.3.1
rootdir: /home/andrii/workspace/robottelo, inifile: 
plugins: xdist-1.13.1, repeat-0.2
gw0 [28] / gw1 [28]
scheduling tests via LoadScheduling
sssssss.s..s.F..............
====================================== FAILURES ======================================
_______________ SettingTestCase.test_positive_update_foreman_url_param _______________
================== 1 failed, 18 passed, 9 skipped in 374.95 seconds ==================

py.test tests/foreman/ui/test_role.py -k 'test_positive_update_org'
================================ test session starts =================================
platform linux2 -- Python 2.7.10, pytest-2.8.7, py-1.4.31, pluggy-0.3.1
rootdir: /home/andrii/workspace/robottelo, inifile: 
plugins: xdist-1.13.1, repeat-0.2
collected 6 items 

tests/foreman/ui/test_role.py .

================= 5 tests deselected by '-ktest_positive_update_org' =================
====================== 1 passed, 5 deselected in 56.82 seconds =======================

py.test tests/foreman/ui/test_product.py -k 'test_positive_update_name'             
================================ test session starts =================================
platform linux2 -- Python 2.7.10, pytest-2.8.7, py-1.4.31, pluggy-0.3.1
rootdir: /home/andrii/workspace/robottelo, inifile: 
plugins: xdist-1.13.1, repeat-0.2
collected 8 items 

tests/foreman/ui/test_product.py .

================ 7 tests deselected by '-ktest_positive_update_name' =================
====================== 1 passed, 7 deselected in 74.44 seconds =======================


py.test tests/foreman/ui/test_partitiontable.py -n 2
================================ test session starts =================================
platform linux2 -- Python 2.7.10, pytest-2.8.7, py-1.4.31, pluggy-0.3.1
rootdir: /home/andrii/workspace/robottelo, inifile: 
plugins: xdist-1.13.1, repeat-0.2
gw0 [7] / gw1 [7]
scheduling tests via LoadScheduling
.......
============================= 7 passed in 165.60 seconds =============================

py.test tests/foreman/ui/test_organization.py -k 'test_positive_create_with_name'
================================ test session starts =================================
platform linux2 -- Python 2.7.10, pytest-2.8.7, py-1.4.31, pluggy-0.3.1
rootdir: /home/andrii/workspace/robottelo, inifile: 
plugins: xdist-1.13.1, repeat-0.2
collected 32 items 

tests/foreman/ui/test_organization.py .

============= 31 tests deselected by '-ktest_positive_create_with_name' ==============
====================== 1 passed, 31 deselected in 34.44 seconds ======================
```

### CODE REVIEW

File: robottelo/ui/partitiontable.py [hunk 1]
```diff
@@ -21,29 +20,27 @@ def _search_locator(self):
...
         """Configures the os family of partition table."""
         if os_family:
-            Select(
-                self.find_element(locators['ptable.os_family'])
-            ).select_by_visible_text(os_family)
+            self.select(locators['ptable.os_family'], os_family)
 
-    def create(self, name, layout=None, os_family=None):
+    def create(self, name, template_path=None, os_family=None,
+               custom_really=None):
```
Comment (oshtaier) [inline robottelo/ui/partitiontable.py]: Why not `custom_really=True`?

File: robottelo/ui/partitiontable.py [hunk 2]
```diff
@@ -21,29 +20,27 @@ def _search_locator(self):
...
-                )
-        else:
+        if self.wait_until_element(locators['ptable.name']) is None:
             raise UIError(
-                'Could not create partition table "{0}"'.format(name)
+                u'Could not create partition table "{0}"'.format(name)
             )
+        self.find_element(locators['ptable.name']).send_keys(name)
+        if template_path:
+            if custom_really is None:
```
Comment (oshtaier) [inline robottelo/ui/partitiontable.py]: Then that if is redundant

File: robottelo/ui/partitiontable.py [hunk 3]
```diff
@@ -51,23 +48,29 @@ def delete(self, name, really=True):
             name,
             really,
             locators['ptable.delete'],
+            drop_locator=locators['ptable.dropdown'],
         )
 
-    def update(self, old_name, new_name=None,
-               new_layout=None, os_family=None):
+    def update(self, old_name, new_name=None, new_template_path=None,
+               new_os_family=None, custom_really=None):
```
Comment (oshtaier) [inline robottelo/ui/partitiontable.py]: Same here and in the factory

File: robottelo/ui/products.py [hunk 4]
```diff
@@ -60,14 +56,12 @@ def update(self, name, new_name=None, new_desc=None,
                 self.click(common_locators['save'])
             if new_gpg_key:
                 self.click(locators['prd.gpg_key_edit'])
-                type_ele = self.find_element(locators['prd.gpg_key_update'])
-                Select(type_ele).select_by_visible_text(new_gpg_key)
+                self.select(locators['prd.gpg_key_update'], new_gpg_key)
                 self.wait_for_ajax()
```
Comment (oshtaier) [inline robottelo/ui/products.py]: That waiter is redundant

File: robottelo/ui/settings.py [hunk 5]
```diff
@@ -45,10 +44,10 @@ def update(self, tab_locator, param_name, value_type, param_value):
         self.click((strategy, value % param_name))
 
         if value_type == 'dropdown':
-            Select(
-                self.find_element(locators['settings.select_value'])
-            ).select_by_value(param_value)
+            self.wait_until_element(locators['settings.select_value'])
```
Comment (oshtaier) [inline robottelo/ui/settings.py]: That waiter is redundant.
Please, fix base class and change:
`element = self.find_element(locator)`
on
`element = self.wait_until_element(locator)`

**Comment by oshtaier** [comment]
ACK pending comments

**Comment by abalakh** [comment]
Comments addressed.
```python
py.test tests/foreman/ui/test_partitiontable.py -n 2                  
============================ test session starts ============================
platform linux2 -- Python 2.7.10, pytest-2.8.7, py-1.4.31, pluggy-0.3.1
rootdir: /home/andrii/workspace/robottelo, inifile: 
plugins: xdist-1.13.1, repeat-0.2
gw0 [7] / gw1 [7]
scheduling tests via LoadScheduling
.......
======================== 7 passed in 165.82 seconds =========================
(env) ~/workspace/robottelo   ui_select ●  py.test tests/foreman/ui/test_setting.py -k 'test_positive_update_authorize_login_delegation_api_param'
============================ test session starts ============================
platform linux2 -- Python 2.7.10, pytest-2.8.7, py-1.4.31, pluggy-0.3.1
rootdir: /home/andrii/workspace/robottelo, inifile: 
plugins: xdist-1.13.1, repeat-0.2
collected 28 items 

tests/foreman/ui/test_setting.py .

======= 27 tests deselected by '-ktest_positive_update_authorize_login_delegation_api_param' =======
================= 1 passed, 27 deselected in 21.63 seconds ==================
```

**Comment by oshtaier** [comment]
ACK

**Comment by elyezer** [comment]
ACK

**Pull request merged**